src/routes/Dashboard/SearchPlateForm.jsx

```diff
@@ -34,29 +36,45 @@ export default function SearchPlateForm({onSearch, onClear}) {
         onClear()
     }
 
+    const [createVehicle, {loading, error}] = useMutation(CreateVehicleMutation);
+
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+        if (inputValue.length === 7) {
+            setShowMessage(false)
+            await createVehicle({
+                variables: {
+                    data: {
+                        plate: inputValue.split(/\s+/).join('')
+                    }
+                }
+            })
+            onSearch(inputValue)
+            setShowButtonDelete(true)
+
+        } else {
+            setShowButtonDelete(false)
+            setShowMessage(true)
+            onClear()
+        }
+    }
+
     return (
         <form className="relative flex flex-col"
-              onSubmit={e => {
-                  e.preventDefault()
-                  if (inputValue.length === 7) {
-                      setShowMessage(false)
-                      onSearch(inputValue)
-                      setShowButtonDelete(true)
-                  } else {
-                      setShowButtonDelete(false)
-                      setShowMessage(true)
-                      onClear()
-                  }
-              }}>
+              onSubmit={handleSubmit}>
             <input
                 className="rounded-md text-center text-6xl text-gray-800 font-bold mb-3 placeholder:text-gray-200 border-gray-100"
                 type="text"
                 placeholder="PLACA"
                 value={inputValue}
                 onChange={handleChange}
                 onKeyDown={handleKeyDown}/>
+
             {showButtonDelete && <PlateDeleteBtn className="absolute -top-3 right-2" onClick={onDeletePlate}/>}
             {showMessage && <ErrorMessage message="Placa incorrecta" className="mb-3"/>}
+            {loading && <ErrorMessage message="Submitting..." className="mb-3"/>}
```

Comment:
showing an **ERROR MESSAGE** when loading, is not a good idea. 

Remove it from here and put the loading state in the button's label.